CppSamples/Analysis/AnalyzeHotspots/AnalyzeHotspots.pro

```diff
@@ -24,7 +24,7 @@ CONFIG += c++17
 # additional modules are pulled in via arcgisruntime.pri
 QT += qml quick
 
-ARCGIS_RUNTIME_VERSION = 300.0.0
+include($$PWD/../../../ArcGISRuntimeVersion.pri)
```

Comment:
I debated renaming the files and variables to just ArcGISVersion, but it would've doubled the amount of files changed, so I didn't think it was worth muddying the review